tools/nuget/OpenTelemetry.Cpp.Dev.nuspec

```diff
@@ -0,0 +1,24 @@
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>OpenTelemetry.Cpp.Dev</id>
+    <version>0.0.0</version>
+    <authors>OpenTelemetry Authors</authors>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <license type="expression">Apache-2.0</license>
+    <licenseUrl>https://licenses.nuget.org/Apache-2.0</licenseUrl>
+    <icon>opentelemetry-icon-color.png</icon>
+    <projectUrl>https://opentelemetry.io/</projectUrl>
+    <description>OpenTelemetry C++ SDK - Development snapshot</description>
+    <tags>Observability OpenTelemetry Monitoring Telemetry Tracing native C++ v140 v141 winapp dyn rt-dyn msvcstl windesktop</tags>
```

Comment:
winapp?

Reply:
cpprest SDK is cross-plat and has this exact set of tags. I think we'd mainly use NuPkg for Windows builds, with Linux relying on `vcpkg` or Bazel.

Reply:
..but I am open to change this to whatever is more appropriate :)

Reply:
These tags are for discoverability of these packages in nuget feed  - So say searching for `dyn` or `rt-dyn` would result in  otel-cpp packages in search result ?

Reply:
I followed Visual Studio guidelines for creating native nuget packages. It refers to cpprest SDK as a good example of a nuget package. I copied the tags from there, combining the tags from OpenTelemetry C# SDK with the (applicable) tags from cpprest SDK, following the recommended best practices. My understanding is that NuGet tooling in Visual Studio IDE somehow parses those tags. The article that I followed is here:
https://docs.microsoft.com/en-us/nuget/guides/native-packages